src/streaming/text/TextController.js

```diff
@@ -282,6 +302,9 @@ function TextController() {
 
         let oldTrackIdx = textTracks.getCurrentTrackIdx();
         if (oldTrackIdx !== idx) {
+            if (allTracksAreDisabled && mediaController) {
+                mediaController.saveTextSettingsDisabled();
```

Comment:
Can we use mediaController.setTrack here instead of saveTextSettingsDisabled?

Reply:
That would complexify mediaController.setTrack